source/designers-guide/best-practice-theme-development/index.md

```diff
@@ -93,6 +93,9 @@ grunt --shopId=1 # optionally specify shopId
 
 ![Grunt Screenshot](grunt-screenshot.png)
 
+### Development Task
+Since Shopware 5.5.4 we have also a development Grunt task. This task will compile LESS and Javascript Files and exits without watching the files.
```

Comment:
Since Shopware 5.5.4 there is a development grunt task, which will compile LESS and Javascript files without starting to watch them afterwards.